packages/@okta/vuepress-site/docs/guides/social-login/main/index.md

```diff
@@ -49,12 +40,13 @@ To add <StackSnippet snippet="idp" inline /> as an Identity Provider in Okta:
 1. Click **Next**.
 1. In the **General Settings** section, define the following:
 
-    * **Name**: Enter a name for the Identity Provider in Okta.
-    * **Client ID**: Paste the generated client ID from your <StackSnippet snippet="idp" inline /> application.
-    * **Client Secret**: Paste the generated client secret from your <StackSnippet snippet="idp" inline /> application.
-    * **Scopes**: Leave the defaults for a simple sign-in flow. You can also add more scopes. See <StackSnippet snippet="scopes" inline />.
+   <StackSnippet snippet="appidpinokta" />
+
+1. Click **Finish**. A page appears that displays the IdP's configuration.
 
-1. Click **Finish**.
+<StackSnippet snippet="afterappidpinokta" />
+
+> **Note:** See the [Identity Providers API](/docs/reference/api/idps/#add-identity-provider) for request and response examples of creating an Identity Provider in Okta using the API.
```

Comment:
would omitting 'request and response' be a bad thing here? Otherwise I wonder if something like:

"...for examples of requests and responses when creating Identity Providers in Okta using the API."?